### PR TITLE
Add support for Facebook login for business configurations

### PR DESCRIPTION
**Pull request by norkans7**
Add support for Facebook login for business
https://developers.facebook.com/docs/facebook-login/facebook-login-for-business

### CODE REVIEW

File: templates/channels/types/facebookapp/refresh_token.html [hunk 1]
```diff
@@ -32,14 +32,29 @@
       FB.init({
         appId: '{{ facebook_app_id }}',
         xfbml: true,
-        version: 'v12.0'
+        version: 'v17.0'
```
Comment (norkans7) [inline templates/channels/types/facebookapp/refresh_token.html:35]: SDK API version was old here but I suspect that is why the refresh token was not working sometimes, I make that match the API version we use to connect the pages

File: templates/channels/types/facebookapp/refresh_token.html [hunk 2]
```diff
@@ -32,14 +32,29 @@
       FB.init({
         appId: '{{ facebook_app_id }}',
         xfbml: true,
-        version: 'v12.0'
+        version: 'v18.0'
```
[the remark this answers is not in the document]
Comment (norkans7) [reply templates/channels/types/facebookapp/refresh_token.html]: I kept the version to v17.0 for now and we'll update to v18.0 at a different time

[the remark this answers is not in the document]
Comment (norkans7) [reply templates/channels/types/facebookapp/refresh_token.html]: v.17.0 has been used for a while now

File: templates/channels/types/facebookapp/refresh_token.html (hunk 1 above)
Comment (rowanseymour) [reply templates/channels/types/facebookapp/refresh_token.html:35]: are we intending to revert to 17 here?

File: templates/channels/types/instagram/refresh_token.html [hunk 3]
```diff
@@ -32,14 +32,15 @@
       FB.init({
         appId: '{{ facebook_app_id }}',
         xfbml: true,
-        version: 'v18.0'
+        version: 'v17.0'
```
Comment (rowanseymour) [inline templates/channels/types/instagram/refresh_token.html:35]: as above

File: templates/channels/types/whatsapp/connect.html [hunk 4]
```diff
@@ -55,7 +55,7 @@
         appId: '{{ facebook_app_id }}', // Meta App ID
         cookie: true, // enable cookies
         xfbml: true, // parse social plugins on this page
-        version: 'v18.0' //Graph API version
+        version: 'v17.0' //Graph API version
```
Comment (rowanseymour) [inline templates/channels/types/whatsapp/connect.html:58]: as above

File: templates/channels/types/facebookapp/refresh_token.html (hunk 1 above)
Comment (norkans7) [reply templates/channels/types/facebookapp/refresh_token.html]: No, mistake while merging

**Pull request merged**